package-lock.json

```diff
@@ -13298,10 +13301,11 @@
       "dev": true
     },
     "woofmark": {
-      "version": "git://github.com/jywarren/woofmark.git#12e2c1d4559268e2de605ad6084d5dce24ea42c6",
+      "version": "git://github.com/jywarren/woofmark.git#39237b798308e6855acde1a938c18a8e4da62fce",
```

Comment:
Only this should had changed, everything else should be same.

Reply:
See this https://stackoverflow.com/questions/44297803/what-is-the-role-of-the-package-lock-json#:~:text=the%20npm%20docs%3A-,package-lock.,regardless%20of%20intermediate%20